test/TestAsset/Constructor/OptionalArguments.php

```diff
@@ -12,7 +12,7 @@
 
 class OptionalArguments
 {
-    public function __construct($foo = null, $bar = 'something')
+    public function __construct(?string $foo = null, string $bar = 'something')
```

Comment:
This changes the constructor signature of this test asset. Consider using doc-block tags instead.